TeamCode/src/main/java/org/firstinspires/ftc/teamcode/GMR/Robot/Robot.java

```diff
@@ -66,7 +68,7 @@ public Robot (HardwareMap hardwareMap, Telemetry telemetry) {
         relicClamp = hardwareMap.servo.get("relicclamp");
 
         rightColor.setPosition(0);
-        leftColor.setPosition(0.85);
+        leftColor.setPosition(0.85);*/
```

Comment:
Please remove commented out code. We have Git history to restore if any of this is needed in the future.